libs/gllm-plugin/gllm_plugin/pipeline/pipeline_handler.py

```diff
@@ -165,37 +164,50 @@ async def acleanup_plugins(cls, instance: "PipelineHandler") -> None:
             instance (PipelineHandler): The handler instance.
         """
         for plugin in instance._plugins.values():
-            await plugin.cleanup()
+            try:
+                await plugin.cleanup()
+            except Exception as e:
+                logger.error(f"Error cleaning up plugin `{plugin.name}`: {e}")
+                pass
 
     @classmethod
     async def _build_plugin(
-        cls, instance: "PipelineHandler", chatbot_id: str, supported_models: list[str], plugin: Plugin
+        cls, instance: "PipelineHandler", chatbot_id: str, supported_models: list[dict[str, Any]], plugin: Plugin
     ) -> None:
         """Build a plugin for the given chatbot.
 
         Args:
             instance (PipelineHandler): The handler instance.
             chatbot_id (str): The chatbot ID.
-            supported_models (list[str]): List of supported models.
+            supported_models (list[dict[str, Any]]): List of supported models.
```

Comment:
```suggestion
            supported_models (list[dict[str, Any]]): List of models (including config).
```
nitpick:
supaya konsisten sama line 45

Reply:
updated di https://github.com/GDP-ADMIN/glchat-sdk/pull/43/commits/c69c45f1b77ab68f2ec76d181034dc707f5dfb4b